SmartDeviceLink/SDLIAPSession.m

```diff
@@ -115,29 +115,29 @@ - (BOOL)isStopped {
 - (void)sendData:(NSData *)data {
     // Enqueue the data for transmission on the IO thread
     [self.sendDataQueue enqueueBuffer:data.mutableCopy];
+
+    [self performSelector:@selector(sdl_dequeueAndWriteToOutputStream) onThread:self.ioStreamThread withObject:nil waitUntilDone:NO];
```

Comment:
Key change in this fix. sdl_dequeueAndWriteToOutputStream will be called in ioStreamThread, therefore this change meets stream handling requirements.